src/scripting/lua_root.cc

```diff
@@ -642,45 +643,171 @@ int LuaDescriptions::new_tribe(lua_State* L) {
-      :const:`"worker"`            :const:`"programs"`       **program_name**   (*string*),
-                                   :const:`"set"`            **actions_table**  (*table*)
-      :const:`"productionsite"`    :const:`"enhancement"`    **building_name**  (*string*)
-      :const:`"productionsite"`    :const:`"input"`          **ware_name**      (*string*),
-                                   :const:`"add_ware"`       **amount**         (*int*)
-      :const:`"productionsite"`    :const:`"input"`          **ware_name**      (*string*),
-                                   :const:`"modify_ware"`    **new_amount**     (*int*)
-      :const:`"productionsite"`    :const:`"input"`          **ware_name**      (*string*)
-                                   :const:`"remove_ware"`
-      :const:`"productionsite"`    :const:`"input"`          **worker_name**    (*string*),
-                                   :const:`"add_worker"`     **amount**         (*int*)
-      :const:`"productionsite"`    :const:`"input"`          **worker_name**    (*string*),
-                                   :const:`"modify_worker"`  **new_amount**     (*int*)
-      :const:`"productionsite"`    :const:`"input"`          **worker_name**    (*string*)
-                                   :const:`"remove_worker"`
-      :const:`"productionsite"`    :const:`"programs"`       **program_name**   (*string*),
-                                   :const:`"set"`            **program_table**  (*table*)
-      :const:`"tribe"`             :const:`"add_ware"`       **ware_name**      (*string*),
-                                                             **menu_column**    (*int*),
-                                                             **target_quan**    (*int* or *nil*),
-                                                             **preciousness**   (*int*),
-                                                             **helptexts**      (*table*)
-      :const:`"tribe"`             :const:`"add_worker"`     **worker_name**    (*string*),
-                                                             **menu_column**    (*int*),
-                                                             **target_quan**    (*int* or *nil*),
-                                                             **preciousness**   (*int* or *nil*),
-                                                             **helptexts**      (*table*)
-      :const:`"tribe"`             :const:`"add_building"`   **building_name**  (*string*),
-                                                             **helptexts**      (*table*)
-      :const:`"tribe"`             :const:`"add_immovable"`  **immovable_name** (*string*),
-                                                             **helptexts**      (*table*)
-      ===========================  ========================  ====================================
+      **Type "resource"**
+
```

Comment:
Format it as a header? According to [ReSt primer](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections) this should work:

~~~~
Type "resource"
^^^^^^^^^^^^^^^
~~~~

Reply:
If I left-align the heading it looks like this:
![grafik](https://user-images.githubusercontent.com/48293446/119234050-0dd5c280-bb2c-11eb-87a9-ebe44147c400.png)
and if I indent it I get `"WARNING: Unexpected section title."`

Reply:
Hmm... what about giving the table a caption and make the needed formatting by css?

~~~~
.. table:: This is the caption
   :widths: …
~~~~

![table_caption](https://user-images.githubusercontent.com/6730556/119235686-0dd9c080-bb34-11eb-885b-80e2aea83ed2.png)

With css we can align it left, make it bigger, or bold, and so on.